src/Cake.Common.Tests/Unit/XML/XmlTransformationTests.cs

```diff
@@ -182,7 +182,7 @@ public void Should_Transform_Xml_String_And_Xsl_String_To_Result_String_With_Xml
                 Assert.Equal(htm, result, ignoreLineEndingDifferences: true);
             }
 
-            //this feels wrong to be CLR only
+            // this feels wrong to be CLR only
```

Comment:
Yes, could this be an .NET Core issue we you raise upstream? Looks like a BOM is received, even if  it's set to false or something with GetString(). Haven't gone in detail, but it does feel weird.